contracts/CTKNCrowdsale.sol

```diff
@@ -138,8 +177,36 @@ contract CTKNCrowdsale is CappedCrowdsale, RefundableCrowdsale, MintedCrowdsale
      *  @param _weiAmount Value in wei to be converted into tokens
      *  @return the wei amount in excess of what was needed to buy a whole number of tokens.
      */
-    function _getOverpaymentAmount(uint256 _weiAmount) internal view returns (uint256) {
+    function _getOverpaymentAmount(uint256 _weiAmount)
+        internal
+        view
+        returns (uint256)
+    {
       return _weiAmount % rate;
     }
```

Comment:
Still think we should extend SafeMath to do _weiAmount.mod(rate)

Reply:
invoking library functions uses more gas than straight up arithmetic as it uses the `delegate_call` opcode under the hood.  I don't see a reason to do that for this.